dss_datamover/utils/config.py

```diff
@@ -255,6 +261,12 @@ def delete(self, subparser):
                                help='Run DataMover in debug mode')
         subparser.add_argument("--profile", "-pro", required=False, action='store_true',
                                help='Profiling of DEL operation (Not Implemented)')
+        subparser.add_argument("--nfs-server", "-ns", type=str, required='--nfs-port' in sys.argv or '--nfs-share' in sys.argv,
+                               help='specify NFS server ip addr')
+        subparser.add_argument("--nfs-port", "-np", type=str, required='--nfs-server' in sys.argv or '--nfs-share' in sys.argv,
+                               help='specify NFS server port')
+        subparser.add_argument("--nfs-share", "-nsh", type=str, required='--nfs-server' in sys.argv or '--nfs-port' in sys.argv,
+                               help='specify NFS share/directory to use')
```

Comment:
We can make nfs-server and nfs-share as mutually dependent. If one is given, then the other one also has to be given